ui/components/AutomationDetail.tsx

```diff
@@ -62,7 +63,7 @@ function AutomationDetail({ automation, className, info }: Props) {
         <SubRouterTabs rootPath={`${path}/details`}>
           <RouterTab name="Details" path={`${path}/details`}>
             <ReconciledObjectsTable
-              automationKind={AutomationKind.KustomizationAutomation}
+              automationKind={automation?.type === AutomationType.Kustomization ? AutomationKind.KustomizationAutomation: AutomationKind.HelmReleaseAutomation}
```

Comment:
This seems like a larger issue of `kind` and `type` being duplicates of each other. Can you create a follow up to fix that?

Reply:
Done https://github.com/weaveworks/weave-gitops/issues/2162